### PR TITLE
Add footer to layout

### PR DESCRIPTION
**Pull request by tobyhede**
Links that don't exist are just text for now.

### CODE REVIEW

**Comment by maxious** [comment]
LGTM

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -28,6 +28,52 @@
       = yield
     %footer{role: "contentinfo"}
       .wrapper
-        %span
-          - unless user_signed_in?
-            = link_to('Sign in', new_user_session_path)
```
Comment (tessereth) [inline app/views/layouts/application.html.haml]: You've removed the sign in link from the footer but not included it anywhere else. We really do need to have it somewhere.

Comment (tobyhede) [inline app/views/layouts/application.html.haml]: Any thoughts on where should it be?

Comment (tobyhede) [inline app/views/layouts/application.html.haml]: @tessereth dan has confirmed no link

Comment (tessereth) [inline app/views/layouts/application.html.haml]: Sure

**Comment by tessereth** [comment]
LGTM

**Pull request merged**